### PR TITLE
fix: Some upgrades erc725.js and some minor npms for security fixes

### PR DESCRIPTION
**Pull request by richtera**
### Description

Upgrades for erc725.js
Upgrades for some minors npms to capture security fixes.

### CODE REVIEW

**Comment by richtera** [comment]
@dzbo NOTE: When I ran the test locally the 404 page was a few pixels off. Not sure what that's about, but I am assuming it's just a local thing, but FYI.

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # neccessary Algolia envs for searching to work properly
-ALGOLIA_APPLICATION_ID=YOUR_ALGOLIA_APPLICATION_ID
-ALGOLIA_API_KEY=YOUR_ALGOLIA_API_KEY
+ALGOLIA_APPLICATION_ID=YHFN1WRCR5
+ALGOLIA_API_KEY=5981bf15ffc7630353478168f2574228
```
Comment (Hugoo) [inline .env.example:3]: ??? Is this supposed to be public?!

**Review by Hugoo — APPROVED** [review]
Yes except for the API keys?!

File: .env.example (hunk 1 above)
Comment (richtera) [reply .env.example:3]: Argh, how did that get in there.

Comment (richtera) [reply .env.example:3]: Fixing it ended up in the example and not in .env.

Comment (richtera) [reply .env.example]: Removed. It's only inside of thie PR note. I removed it from git history and the repo is not public.

Comment (richtera) [reply .env.example]: It's actually a public key for search anyways, but better not in the repo.

**Review by dzbo — APPROVED** [review]

**Pull request merged**